documentation/modules/exploit/unix/webapp/thinkphp_rce.md

```diff
@@ -118,30 +115,27 @@ meterpreter >
 ### ThinkPHP 5.0.23 from [Vulhub](https://github.com/vulhub/vulhub/tree/master/thinkphp/5.0.23-rce)
 
 ```
-msf5 exploit(unix/webapp/thinkphp_rce) > set rport 8081
```

Comment:
Same `RPORT` this time, just swapping containers. The tested target runs on 8080/TCP.